agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -621,13 +626,13 @@ func telemetryTestWithStatsPolling(t *testing.T, taskDefinition string) {
 	params.EndTime = aws.Time(RoundTimeUp(time.Now(), time.Minute).UTC())
 	params.StartTime = aws.Time((*params.EndTime).Add(-waitMetricsInCloudwatchDuration).UTC())
 	params.MetricName = aws.String("CPUUtilization")
-	metrics, err := VerifyMetrics(cwclient, params, false)
+	metrics, err := VerifyMetrics(cwclient, params, false, 0.0)
 	assert.NoError(t, err, "Task is running, verify metrics for CPU utilization failed")
 	// Also verify the cpu usage is around expectedCPUPercentage +/- 5%
-	assert.InDelta(t, expectedCPUPercentage*100.0, *metrics.Average, 5)
+	assert.InDelta(t, expectedCPUPercentage*100.0, *metrics.Average, statsNoiseDelta)
 
 	params.MetricName = aws.String("MemoryUtilization")
-	metrics, err = VerifyMetrics(cwclient, params, false)
+	metrics, err = VerifyMetrics(cwclient, params, false, 0.0)
```

Comment:
why is this one zero instead of statsNoiseDelta?

Reply:
golang won't let me overload a function/use variadic args.

The existing `VerifyMetrics` signature uses the `idleCluster` bool:
```
func VerifyMetrics(cwclient *cloudwatch.CloudWatch, params *cloudwatch.GetMetricStatisticsInput, idleCluster bool)
```
So now if `idleCluster = true`, we need a corresponding noiseDelta float.  There are no checks for `idleCluster = false` so the 0.0 is effectively a nil placeholder wherever we have `false`.  If there's a better way of doing this, I'm open to it.

Reply:
Can we use statsNoiseDelta as a constant within the function? I don't think we gain any flexibility by passing it around.

Reply:
updated.